lib/Serialization/ModuleFormat.h

```diff
@@ -2,7 +2,7 @@
 //
 // This source file is part of the Swift.org open source project
 //
-// Copyright (c) 2014 - 2018 Apple Inc. and the Swift project authors
+// Copyright (c) 2014 - 2020 Apple Inc. and the Swift project authors
```

Comment:
(minor, if you care -- technically 2021, if you're using some script to update maybe change in it?)